types/seeed/senseCAPS2105/default.schema.json

```diff
@@ -9,25 +9,38 @@
     "soilHumidity": {
       "title": "Soil humidity",
       "unit": "%",
-      "type": "number",
+      "type": [
+        "number",
+        "null"
+      ],
       "description": "The relative humidity of the soil in %",
       "minimum": 0,
       "maximum": 100
     },
     "soilTemperature": {
       "title": "Soil temperature",
       "unit": "°C",
-      "type": "number",
+      "type": [
+        "number",
+        "null"
+      ],
       "description": "The temperature in °C.",
       "minimum": -3276.5,
       "maximum": 3276.5
     },
     "soilConductivity": {
       "title": "Soil conductivity",
       "unit": "uS/cm",
-      "type": "number",
+      "type": [
```

Comment:
this is currently not supported by the backend, please revert it and make a TODO to add it later